bin/MessagesBundle_ja.properties

```diff
@@ -115,6 +115,22 @@ Restart=起動
 Drawing=描画
 FidoCad=FidoCadJ拡張
 pcbline_width = PCB配線幅:
+
+Theme_management=Theme and color management
```

Comment:
Put all the untranslated resources at the end of the file. They are easier to find for those who would like to work on them.